tools/dev-doctor/rootcmd.go

```diff
@@ -147,6 +147,28 @@ func rootCmdRun(cmd *cobra.Command, args []string) {
 			versionRegexp: regexp.MustCompile(`pip (\d+\.\d+\S*)`),
 		},
 		dockerGroupCheck{},
+		&binaryCheck{
+			name:          "golangci-lint",
```

Comment:
The check for `golangci-lint` is already present above, is there any point in re-adding it here or is it a copy-paste leftover?

Reply:
this is clearly the mistake from my side :(, I should have taken a quick look on diff before marking PR ready.